hack/config.sh

```diff
@@ -1,8 +1,18 @@
-binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl cmd/virt-manifest"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirt-kubevirt images/spice-proxy cmd/virt-migrator"
-docker_prefix=kubevirt
-docker_tag=${DOCKER_TAG:-latest}
-manifest_templates="`ls manifests/*.in`"
-master_ip=192.168.200.2
-master_port=8184
-network_provider=weave
+unset binaries docker_images docker_prefix docker_tag manifest_templates \
```

Comment:
Yeah, I made sure that only non-expanded variables are used in the Vagrantfile.

At the moment we only use master_ip and network_provider. Just a solution for these two variables might also be an option ...